frontend/src/app/tasks-page/tasks-page.component.ts

```diff
@@ -13,14 +17,37 @@ export class TasksPageComponent implements OnInit {
 
   page: Page<TaskModel>;
 
-  constructor(private route: ActivatedRoute, private router: Router) {
+  constructor(private route: ActivatedRoute,
+              private router: Router,
+              private taskService: TaskService,
+              private tasksResolverService: TasksResolverService) {
   }
 
   ngOnInit(): void {
     this.route.data.subscribe(data => this.page = data.tasks);
   }
 
-  loadPage(pageNumber: number) {
-    this.router.navigate(['.'], {relativeTo: this.route, queryParams: {page: (pageNumber - 1).toString()}})
+  loadPage(viewPageNumber: number) {
+    this.router.navigate(['.'], {relativeTo: this.route, queryParams: {page: (viewPageNumber - 1).toString()}})
+  }
+
+  onTaskClicked(event: TaskEvent) {
```

Comment:
I'm all for clarifying with comments, or even better: a better name and comments. Should I rename it "onTaskActioned", or "onTaskAction", or "onTaskEvent"? What would you like to see as comment? Something like

> Method called from the template when a TaskEvent is emitted by the child TasksComponent, indicating that one of the action buttons (assign, edit, cancel, etc.) of a task has been clicked.